src/napari_builtins/_qt/features_table.py

```diff
@@ -393,34 +416,37 @@ def __init__(
         viewer: napari.viewer.ViewerModel,
     ) -> None:
         super().__init__()
-        self._active_layer = None
+        self._selected_layers = []
         self._selection_blocked = False
 
         self.viewer = viewer
-        self.viewer.layers.selection.events.active.connect(
-            self._on_active_layer_change
+        self.viewer.layers.selection.events.changed.connect(
+            self._on_layer_selection_change
         )
 
         self.setLayout(QVBoxLayout())
 
         self.info = QLabel('')
         self.toggle = QToggleSwitch('editable.')
+        self.join_toggle = QToggleSwitch('common_columns.')
```

Comment:
To me "shared" has a more expected meaning than "common". The latter implies a "usual"-ness to the columns, rather than their shared nature
```suggestion
        self.toggle = QToggleSwitch('editable')
        self.join_toggle = QToggleSwitch('shared columns')
```